docs/developers/releasing.rst

```diff
@@ -148,7 +148,7 @@ quality checks.
 
 .. code-block:: bash
 
-    $ pootle regenerate_checks_descriptions
+    $ DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates
```

Comment:
I mean, can't it be split into two commands, one for setting the envvar and another to run the command?